### PR TITLE
2022 Sustainability Chapter

### LINKED ISSUE #2910: Sustainability 2022
# Sustainability 2022

If you're interested in contributing to the Sustainability chapter of the 2022 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide).

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| ---- | ------- | --------- | -------- | ------- | ----------- |
| @ldevernay | @ldevernay @gerrymcgovernireland @timfrick  | @mrchrisadams @cqueern @Djohn12 | @fershad @camcash17 @4upz  | - | @tunetheweb |

<details>
<summary><b>Expand for more information about each role 👀</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [**0. Form the content team**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
+ [X] **May 1**: The content team has at least one author, reviewer, and analyst

### [**1. Plan content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
+ [X] **May 15** The content team has completed the chapter outline in the [draft doc][~google-doc]

### [**2. Gather data**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
+ [x] **June 1**: Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/custom-metrics/blob/main/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress
- **June 1 - 15**: HTTP Archive runs the June crawl

### [**3. Validate results**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
+ [x] **August 1**: Analysts have queried all metrics and saved the output to the [results sheet][~google-sheets]

### [**4. Draft content**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
+ [x] **September 1**: The content team has written, reviewed, and edited the chapter in the [doc][~google-doc]

### [**5. Publication**](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
+ [ ] **September 15**: The completed chapter and all required [metadata](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters) and [figures](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Figures-Guide) are converted to markdown and submitted to [GitHub][~chapter-markdown]
- **September 26**: Target launch date 🚀

## Chapter resources

Refer to these 2022 Sustainability resources throughout the content creation process:

[📄 Google Docs][~google-doc] for outlining and drafting content
[🔍 SQL files][~sql-files] for committing the queries used during analysis
[📊 Google Sheets][~google-sheets] for saving the results of queries
[📝 Markdown file][~chapter-markdown] for publishing content and managing public metadata
[💬 #web-almanac-sustainability on Slack][~slack-channel] for team coordination

[~google-doc]: https://docs.google.com/document/d/1g1ACWRTAzTlcaKKODNASLkXe4zquF-XR5oHh0GfayP8/edit?usp=sharing
[~sql-files]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2022/sustainability
[~google-sheets]: https://docs.google.com/spreadsheets/d/1wU3SjB8XYkbaqxYt8CNtbmDbjCcYZ8m5kiYof7uyI5k/edit?usp=sharing
[~chapter-markdown]: https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/src/content/en/2022/sustainability.md
[~slack-channel]: https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA

### PR DESCRIPTION
**Pull request by Djohn12**
2022 sustainability chapter pull request

Closes #2910 

Staged link: https://20220915t104102-dot-webalmanac.uk.r.appspot.com/en/2022/sustainability
Featured-Quote: https://20220915t104102-dot-webalmanac.uk.r.appspot.com//en/2022/?feat=sustainability#featured-chapter

### CODE REVIEW

File: src/content/en/2022/sustainability.md [hunk 1]
```diff
@@ -1,22 +1,814 @@
...
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/
 title: Sustainability
-#TODO - Review and update chapter description
-description: Sustainability chapter of the 2022 Web Almanac covering ...
-authors: [ldevernay, gerrymcgovernireland, tpgreenwood, timfrick]
-reviewers: [mrchrisadams, cqueern, Djohn12, hanopcan]
+description: Sustainability chapter of the 2022 Web Almanac covering environmental impacts of web pages, where they come from and how to reduce them
+authors: [ldevernay, gerrymcgovernireland, timfrick]
+reviewers: [mrchrisadams, cqueern, Djohn12]
 analysts: [fershad, camcash17, 4upz]
```
Comment (tunetheweb) [inline src/content/en/2022/sustainability.md:7]: @ldevernay can you please check all the contributors are listed in each section (in descending order of contribution) and that we haven't forgotten anyone?

**Review by tunetheweb — APPROVED** [review]
@rviscomi I think this is good to merge, but let me know if you spot anything that needs fixing.

**Comment by rviscomi** [comment]
I haven't reviewed but if it LGTY let's merge

File: src/content/en/2022/sustainability.md (hunk 1 above)
Comment (ldevernay) [reply src/content/en/2022/sustainability.md:7]: Already checked that, we're OK!

**Pull request merged**